### ISSUE TITLE
[Keyboard navigation - Make Code - New Project (Editor)] 'Make a function' control is not accessible through keyboard under New Project (Editor).

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2368]
**User Experience:**
User rely on keyboard will find difficulty to interact with the control if that control is not accessible through keyboard and won't be able to perform functionality over such control.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Functions" controls appearing under Advance section and select it.
6. Navigate to 'Make a function' control.
7. Verify 'Make a function' control by using keyboard.

**Actual Result:**
'Make a function' control is not accessible through keyboard under New Project (Editor) and due to this, user is unable to interact with these controls and won’t be able to perform functionality over such controls.

**Expected Result:**
'Make a function' control should be accessible through keyboard under New Project (Editor) so that user could easily interact with these controls and could perform functionality over these controls.

**Note:**
This issue also exists with Chrome.

**MAS Reference:**
[MAS 2.1.1 - Keyboard](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b8492c4eb-c179-40ae-8777-cd044ed725a2%7d)

**Labels**
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS2.1.1, #Win10-Chrome, #Win10-Edge, #accessibility

[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.)

[MAS2.1.1_New Project (Editor).zip](https://github.com/microsoft/pxt-microbit/files/3476766/MAS2.1.1_New.Project.Editor.zip)

**Comment by jwunderl** [comment]
This is a valid issue that should be fixed eventually, but only applies to the blocks editor which is not currently part of the accessible coding experience re: https://makecode.com/accessibility

**Comment by abchatra** [comment]
Blocks editor has an exception and we are working with Google blockly team to build and accessible story for blocks.
